### ISSUE TITLE
`Insert Code Cell` should put the cursor straight into the code cell if we "remember" the previous language

### ISSUE THREAD

**Issue by DavisVaughan** [quarto-dev/quarto#539]
I like the Quarto feature that "remembers" that I've chosen R as my language of choice in other cells, so when I open a new code cell it defaults to R.

I think we should lean into this feature even further by assuming that the user wants to use R, and put their cursor directly into the code cell rather than into the language selector.

If they want to switch languages, they can go back up and manually change the language name, but I think switching languages is the 1% case, vs the 99% case being continuing usage of the same language (so we should make the 99% case as seamless as possible).

https://github.com/user-attachments/assets/df9ae9e8-ac7a-4ba8-815d-e06e96e57838

In other words, I want it to mostly work like RStudio once we have "remembered" a language choice from the user

https://github.com/user-attachments/assets/2f43aed2-ef94-4827-a8f4-e1d317ab963a

**Comment by DavisVaughan** [comment]
Ah this is https://github.com/quarto-dev/quarto/issues/425
